egs2/TEMPLATE/svs1/README.md

```diff
@@ -281,35 +281,36 @@ $  --pretrained_model /exp/xiaoice-2-24-250k/500epoch.pth:svs:svs.fftsinger \
 
 
 ### VISinger training
```

Comment:
May be we can change the title to `VISinger (1+2)` to note that we support both versions?